app/components/agenda/agendaitem/agendaitem-case-panel/agendaitem-case-panel-view.hbs

```diff
@@ -104,8 +108,14 @@
         <Auk::Toolbar as |Toolbar|>
           <Toolbar.Group @position="left" as |Group|>
             <Group.Item>
-              <pre class="auk-u-text-pre-line" data-test-agendaitem-titles-comment>
-                <strong class="auk-u-text-bold">{{t "comment-title"}}: </strong>{{@agendaitem.comment}}
+              <pre
+                class="auk-u-text-pre-line"
+                data-test-agendaitem-titles-comment
+              >
+                <strong class="auk-u-text-bold">
+                  {{t "comment-title"}}:
+                </strong>
+                {{@agendaitem.comment}}
```

Comment:
reason why: look at the failed test screenshots, there is a lot of added whitespace to both comment and private-comment because the line got formatted